packages/core/components/DirectoryTree/index.tsx

```diff
@@ -81,11 +80,9 @@ export default function DirectoryTree(props: FileListProps) {
                 aria-multiselectable="true"
                 id={Tutorial.FILE_LIST_ID}
             >
-                {!error && (!content || (Array.isArray(content) && !content.length)) && (
-                    <EmptyFileListMessage />
```

Comment:
Does this get rendered anywhere still? It seems like now if a user creates a query that gives no results, there's just a blank screen w/ no message

Reply:
Yea the `FileList` will generate this, this one was a duplicate usage from what I could tell

Reply:
I think it's not rendering for me locally with [groups, e.g. this one](http://localhost:9001/app?group=638+Laser+Power&filter=%7B%22name%22%3A%22638+Laser+Power%22%2C%22value%22%3A%221.39mW%22%7D&filter=%7B%22name%22%3A%22Cell+Line%22%2C%22value%22%3A%22AICS-0122%22%7D&openFolder=%5B%221.39mW%22%5D&source=%7B%22name%22%3A%22imaging_and_segmentation_data.csv+%288%2F15%2F2024+4%3A26%3A03+PM%29%22%2C%22type%22%3A%22csv%22%2C%22uri%22%3A%22https%3A%2F%2Fallencell.s3.amazonaws.com%2Faics%2Femt_timelapse_dataset%2Fmanifests%2Fimaging_and_segmentation_data.csv%3FversionId%3DWmTjARBNL4rNJhV4N7YFYr2dKHWlCHwc%22%7D&sourceMetadata=%7B%22name%22%3A%22Imaging_and_segmentation_data_column_description.csv+%288%2F15%2F2024+4%3A26%3A02+PM%29%22%2C%22type%22%3A%22csv%22%2C%22uri%22%3A%22https%3A%2F%2Fallencell.s3.amazonaws.com%2Faics%2Femt_timelapse_dataset%2Fmanifests%2FImaging_and_segmentation_data_column_description.csv%3FversionId%3D.bmbr.UUT06F9nupeuwxVBYuTMyKyYu6%22%7D)

Reply:
Dang, I thought this was an excessive entry, but it seems it is used when the DirectoryTree is rendering an empty list of tree nodes (folders). The excessive render I was trying to remove is from the `!content` half of the boolean clause which is caused by the totalCount of a file list being `0` or `null` which happens while we are preparing a data source. I added this back, but removed the `!content` half of the boolean.